Wikipedia/Code/TalkPageContainerViewController.swift

```diff
@@ -4,12 +4,12 @@ import UIKit
 @objc(WMFTalkPageContainerViewController)
 class TalkPageContainerViewController: ViewController, HintPresenting {
     
-    private let talkPageTitle: String
-    private let siteURL: URL
-    private let type: TalkPageType
+    let talkPageTitle: String
+    let siteURL: URL
+    let type: TalkPageType
     private let dataStore: MWKDataStore
     private let controller: TalkPageController
-    private let talkPageSemanticContentAttribute: UISemanticContentAttribute
+    let talkPageSemanticContentAttribute: UISemanticContentAttribute
```

Comment:
Doesn't seem like this is used outside anywhere, I'll test to see if it affects anything. Might need to add to the Info struct.

Reply:
it was needed to create `TalkPageReplyListViewController` initially but now I'm using `pushToReplyThread` so doesn't need to be public anyone 👍